### PR TITLE
Add Korean translation for #47

### PR DESCRIPTION
**Pull request by HyukjinKwon**
This PR proposes to add the Korean translation for #47.

### CODE REVIEW

File: README-KO.md [hunk 1]
```diff
@@ -127,8 +127,7 @@ Scala는 매우 강력하며 여러가지 페러다임에 적용 가능한 언
   val clientPort = 2000
   ```
 
-- 지엽적인 공간에서 변수 이름이 하나의 글자로 명명 되는 것은 괜찮습니다. 예를 들어, "i" 는 길지 않은 순환문 에서 (예를 들어, 10 라인의 코드) 그 순환문 안에서의 인덱스를 나타내기 위해 자주 사용 됩니다. 그러나, "l" (Larry의 맨 앞자)를 식별자로 사용하지 않습니다. 왜냐하면, "l", "1", "|" 그리고 "I" 은 구분하기가 매우 어렵기 때문 입니다.
-
+- 지엽적인 공간에서 변수 이름이 하나의 글자로 명명 되는 것은 괜찮습니다. 예를 들어, "i" 는 길지 않은 순환문 에서 (예를 들어, 10 라인의 코드) 그 순환문 안에서의 인덱스를 나타내기 위해 자주 사용 됩니다. 그러나, "l" (Larry의 맨 앞자)를 식별자로 사용하지 않습니다. 왜냐하면, "l", "1", "|" 그리고 "I" 은 구분하기가 어렵기 때문 입니다.
```
Comment (HyukjinKwon) [inline README-KO.md]: "매우" meaning "very" was taken out in this line.

**Comment by HyukjinKwon** [comment]
cc @rxin and @yunpark93

**Comment by HyukjinKwon** [comment]
(gentle ping @yunpark93)

**Comment by yunpark93** [comment]
LGTM cc @rxin

**Comment by rxin** [comment]
Thanks. Merging in.

**Pull request merged**